libs/cx/nested-list/nested-list/nested-list.component.ts

```diff
@@ -128,8 +128,6 @@ export class NestedListComponent implements AfterContentInit, NestedListInterfac
 
     /** @hidden */
     private _handleNestedLevel(): void {
-        this._nestedItemsChangesSubscription.unsubscribe();
```

Comment:
you have to fully remove _nestedItemsChangesSubscription logic and have to implement takeUntilDestory operator.
remove this code:
_nestedItemsChangesSubscription declaration,
this._nestedItemsChangesSubscription.add(itemChangesSub),
ngOnDestroy.